### ISSUE TITLE
iframe height calculations cause iframe to get too big

### ISSUE THREAD

**Issue by VilliGunn** [datocms/plugins-sdk#23]
```
"datocms-plugin-sdk": "0.7.2",
"datocms-react-ui": "0.7.2",
"react": "18.2.0",
"react-dom": "18.2.0"
```
It seems there is an issue with the iframe height calculations. When I have an element with a max height set and an overflow within the iframe accounts for the height of the overflowing element.

This happens because the `getMaxScrollHeight` ([here](https://github.com/datocms/plugins-sdk/blob/master/packages/sdk/src/connect.ts#L354)) is looping through all elements within the `body` and returns the larges `scrollHeight` and in some cases that is an overflowing element.

See recording:

https://user-images.githubusercontent.com/17431581/217194120-75a2a707-b479-40b3-97ba-35d8bc40b42c.mov

**Comment by stefanoverna** [comment]
@VilliGunn sorry for the delay.. can you try to use `datocms-plugin-sdk@0.7.3-alpha.0` and see if it's better? we've changed the autoresize strategy, thanks!

**Comment by VilliGunn** [comment]
> @VilliGunn sorry for the delay.. can you try to use `datocms-plugin-sdk@0.7.3-alpha.0` and see if it's better? we've changed the autoresize strategy, thanks!

@stefanoverna 

Nice, this seems to work! Great job 🎉

**Comment by stefanoverna** [comment]
Released as datocms-plugin-sdk@0.7.3!
